src/uct/ib/rc/base/rc_iface.c

```diff
@@ -26,7 +26,7 @@ static const char *uct_rc_fence_mode_values[] = {
 };
 
 ucs_config_field_t uct_rc_iface_common_config_table[] = {
-  {UCT_IB_CONFIG_PREFIX, "RX_INLINE=64;RX_QUEUE_LEN=4095;SEG_SIZE=8256", NULL,
+  {UCT_IB_CONFIG_PREFIX, "RX_INLINE=64;TX_INLINE_RESP=64;RX_QUEUE_LEN=4095;SEG_SIZE=8256", NULL,
```

Comment:
why removed?

Reply:
per offline talk: before the change, TX_INLINE_RESP was taking value from RX_INLINE